samples/bluetooth/cap_acceptor/prj.conf

```diff
@@ -1,6 +1,9 @@
 CONFIG_BT=y
 CONFIG_LOG=y
 CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_OBSERVER=y
```

Comment:
Why do we need the observer here?

Reply:
Just for the BSIM test with `CONFIG_SAMPLE_BROADCAST=y`, alternatively `select BT_OBSERVER` can be added to the `config SAMPLE_BROADCAST` in `cap_acceptor/Kconfig`.

Reply:
Ah, that makes sense. 

It should however be moved to where the Kconfig options for broadcast is enabled: https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/cap_acceptor/Kconfig#L21-L32

Reply:
Yes, I mentioned the same place in my comment above! I’ll move it there.